Podfile.lock

```diff
@@ -45,18 +45,20 @@ SPEC REPOS:
     - Swinject
     - TransitionButton
     - Typhoon
-  https://repo.onegini.com/artifactory/api/pods/cocoapods-public:
-    - OneginiSDKiOS
+
+EXTERNAL SOURCES:
+  OneginiSDKiOS:
+    :path: "~/Projekty/sdk-ios"
```

Comment:
this should not be local.
To make it work you need to get rid of local path, pod deintegrate (possible remove Podfile.lock) and pod install